guides/v2.3/graphql/send-request.md

```diff
@@ -1,31 +1,16 @@
 ---
 group: graphql
-title: GraphQL request headers
+title: GraphQL requests
 ---
 
+In Magento 2.3.1, you must send all queries and mutations as HTTP POST requests. 
```

Comment:
Do we want to be version specific here? Do we anticipate this changing?

Reply:
I specifically mentioned 2.3.1 because this is changing for 2.3.2. I also want anyone looking at the GraphQL repo to immediately realize which version I'm documenting here.